src/index.js

```diff
@@ -1,27 +1,57 @@
 // 1. Write a function that accepts an array of numbers. Return a new array with all elements doubled
 // Ex: double([1,2,3]) => [2,4,6]
 
+const double = (arr) => arr.map((item) => item * 2)
+
 // 2. Write a function that accepts an array of numbers. Return a new array with only elements that are even number
 // Ex: filterEven([1,2,3,4,5,6]) => [2,4,6]
 
+const filterEven = (arr) => arr.filter((item) => item % 2 === 0)
+
+filterEven ([2,3,4,44,55,66,77,88])
```

Comment:
Should be `filterEven([2,3,4,44,55,66,77,88])` with no space between the function name and parenthesis